WGLMakie/src/particles.jl

```diff
@@ -181,7 +181,8 @@ function scatter_shader(scene::Scene, attributes, plot)
 
     if uniform_dict[:shape_type][] == 3
         atlas = wgl_texture_atlas()
-        uniform_dict[:distancefield] = NoDataTextureAtlas(size(atlas.data))
+        # uniform_dict[:distancefield] = NoDataTextureAtlas(size(atlas.data))
+        uniform_dict[:distancefield] = Sampler(map(x -> wgl_texture_atlas().data, TEXTURE_ATLAS))
```

Comment:
This should do pretty much the same as before, just with 10-100x the data transfer, since the texture atlas is huge and lots of plots will then have separate observables which update on write.
The question is, did this actually actually fix it? If yes, I think this must be some kind of update order bug, and this fix changes the order.